compat/history-stream.js

```diff
@@ -50,17 +50,19 @@ exports.init = function (sbot, config) {
 
     return pull(
       pullCont(function (cb) {
-        if (!ref.isFeed(opts.id)) return cb(opts.id + ' is not a feed')
+        sbot.db.log.onDrain(() => {
```

Comment:
onDrain was needed here because we're not using `query` from `db.js` which has the implicit onDrain. Notice I had to export the log in `sbot.db.log`